templates/bundles/TwigBundle/Exception/error.html.twig

```diff
@@ -0,0 +1,22 @@
+{# templates/bundles/TwigBundle/Exception/error.html.twig #}
+{% extends 'base.html.twig' %}
+{% block title 'Page non trouvée - GameHer' %}
+{% block body_id 'error' %}
+
+{% block main %}
+	<div class="error__wrapper">
+		<div class="container">
+				<div class="col error__texts">
+					<h1 class="error__text">
+						<span class="error__text__title">{{ 'error.title' |trans }}</span>
+						<span class="error__text__code">{{ status_code }}</span>
+					</h1>
+						<a class="btn roundBorder error__button " href="{{ path('index') }}" rel="home">{{ 'error.button.text' |trans }}</a>
+
+				</div>
+				<div class="error__illustration">
```

Comment:
I don't see any CSS in the "errors.css" file about this class, is this normal?

Reply:
There isn't, I wrote it expecting to style the element, but it wasn't necessary. 
I removed the class.